library/src/nvidia_detail/hipsolver_sparse.cpp

```diff
@@ -373,4 +373,153 @@ catch(...)
+                                       const int*                csrRowPtr,
+                                       const int*                csrColInd,
+                                       const double*             b,
+                                       double                    tolerance,
+                                       int                       reorder,
+                                       double*                   x,
+                                       int*                      singularity)
+try
+{
+    if(!handle)
+        return HIPSOLVER_STATUS_NOT_INITIALIZED;
+    if(!descrA)
+        return HIPSOLVER_STATUS_INVALID_VALUE;
+
+    return hipsolver::cuda2hip_status(cusolverSpDcsrlsvqr((cusolverSpHandle_t)handle,
+                                                          n,
+                                                          nnzA,
+                                                          (cusparseMatDescr_t)descrA,
+                                                          csrVal,
+                                                          csrRowPtr,
+                                                          csrColInd,
+                                                          b,
+                                                          tolerance,
+                                                          reorder,
+                                                          x,
+                                                          singularity));
+}
+catch(...)
+{
+    return hipsolver::exception2hip_status();
+}
+
+hipsolverStatus_t hipsolverSpCcsrlsvqr(hipsolverSpHandle_t       handle,
+                                       int                       n,
+                                       int                       nnzA,
+                                       const hipsparseMatDescr_t descrA,
+                                       const hipFloatComplex*    csrVal,
+                                       const int*                csrRowPtr,
+                                       const int*                csrColInd,
+                                       const hipFloatComplex*    b,
+                                       double                    tolerance,
+                                       int                       reorder,
+                                       hipFloatComplex*          x,
+                                       int*                      singularity)
+try
+{
+    if(!handle)
+        return HIPSOLVER_STATUS_NOT_INITIALIZED;
+    if(!descrA)
+        return HIPSOLVER_STATUS_INVALID_VALUE;
+
+    return hipsolver::cuda2hip_status(cusolverSpCcsrlsvqr((cusolverSpHandle_t)handle,
+                                                          n,
+                                                          nnzA,
+                                                          (cusparseMatDescr_t)descrA,
+                                                          csrVal,
```

Comment:
If I remember correctly, hipFloatComplex* and hipDoubleComplex* need to be cast to cuComplex* and cuDoubleComplex* when passing to cuSOLVER.